build.rs

```diff
@@ -370,6 +370,8 @@ fn assimp_dynamic_linking() -> (String, String, String) {
 
     let include_path = if target.contains("apple") {
         "/opt/homebrew/opt/assimp/include"
+    } else if std::env::var("DOCS_RS").is_ok() {
```

Comment:
hey man, who is supposed to set this env var? Which value should it have?
Also, why is that an else if clause on the include path?
:)

Reply:
the env var is set by docs.rs as per this documentation: https://docs.rs/about/builds#detecting-docsrs

it's an `else if` on the include path because it points to the local cloned assimp/ directory from the git submodule instead of a locally installed assimp version. unless you mean why it's not the first fork? i assumed that the target wouldn't be apple because the docs.rs build environment uses linux, but i guess making it the first fork is more correct :sweat_smile: